pubspec.yaml

```diff
@@ -37,3 +37,15 @@ dev_dependencies:
   test: ^1.3.0
   test_descriptor: ^1.0.0
   test_process: ^1.0.0
+
+dependency_overrides:
```

Comment:
Please add a comment explaining the logic here..

Reply:
Done